themes/scss/widgets/chart.scss

```diff
@@ -77,7 +77,7 @@
 
 .CDB-Chart-axis {
   @include css3-prefix(user-select, none);
-  font: $sFontSize-smaller sans-serif;
+  font: $sFontSize-small sans-serif;
```

Comment:
Name of variable `sFontSize-small` should be written in all lowercase letters with hyphens instead of underscores